### PR TITLE
scrape network policies

### PR DESCRIPTION
**Pull request by aojea**
Copy and paste from ScrapeAnet, this is the install manifest

https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/kube-network-policies/kube-network-policies.yaml

/assign @dlapcevic @wojtek-t

### CODE REVIEW

File: clusterloader2/pkg/config/cluster.go [hunk 1]
```diff
@@ -88,6 +88,7 @@ type PrometheusConfig struct {
 	ScrapeMetricsServerMetrics bool
 	ScrapeNodeLocalDNS         bool
 	ScrapeAnet                 bool
+	ScrapeNetPol               bool
```
Comment (wojtek-t) [inline clusterloader2/pkg/config/cluster.go:91]: nit: ScrapeNetworkPolicies [let's avoid shortcuts, especially in configs]

**Review by wojtek-t — COMMENTED** [review]
Just one minor nit, other than that LGTM

File: clusterloader2/pkg/config/cluster.go (hunk 1 above)
Comment (aojea) [reply clusterloader2/pkg/config/cluster.go]: done

**Pull request merged**